pkg/route/ingress/ingress.go

```diff
@@ -514,7 +514,7 @@ func newRouteForIngress(
 			Labels:       ingress.Labels,
 			Annotations:  ingress.Annotations,
 			OwnerReferences: []metav1.OwnerReference{
-				{APIVersion: "extensions/v1beta1", Kind: "Ingress", Controller: &t, Name: ingress.Name, UID: ingress.UID},
+				{APIVersion: "networking.k8s.io/v1beta1", Kind: "Ingress", Controller: &t, Name: ingress.Name, UID: ingress.UID},
```

Comment:
what about existing objects with this owner ref?

Reply:
> what about existing objects with this owner ref?

They will still be fine until the old API is gone entirely, so you just have to scan for and update them.